extension/refreshable_file/ext_system.go

```diff
@@ -0,0 +1,48 @@
+package refreshable_file
+
+import (
+	"errors"
+	"fmt"
+	"github.com/alibaba/sentinel-golang/core/base"
+	"github.com/alibaba/sentinel-golang/core/system"
+	"github.com/tidwall/gjson"
+)
+
+func SystemRulesConvert(src []byte) interface{} {
+	if src == nil {
+		return nil
+	}
+	rs := gjson.ParseBytes(src).Array()
```

Comment:
This is a little problem: what should I do if the data souce is a file in toml format?